app/src/main/java/com/fstyle/androidtrainning/widget/customView/TouchEventView.java

```diff
@@ -0,0 +1,82 @@
+package com.fstyle.androidtrainning.widget.customView;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.Path;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.view.View;
+
+/**
+ * Created by DaoLQ on 20/04/2017.
+ */
+
+public class TouchEventView extends View {
+
+    private Paint mPaint;
+    private Path mPath;
+
+    public TouchEventView(Context context) {
+        this(context, null);
+    }
+
+    public TouchEventView(Context context, @Nullable AttributeSet attrs) {
+        this(context, attrs, 0);
+    }
+
+    public TouchEventView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        setAttribute();
+    }
+
+    private void setAttribute() {
+        mPaint = new Paint();
+        mPath = new Path();
+
+        mPaint.setAntiAlias(true);
+        mPaint.setStrokeWidth(10f);
+        mPaint.setColor(Color.BLACK);
+
+        mPaint.setStyle(Paint.Style.STROKE);
+        mPaint.setStrokeJoin(Paint.Join.ROUND);
+    }
+
+    @Override
+    protected void onDraw(Canvas canvas) {
+        canvas.drawRect(0, 0, getWidth(), getHeight(), mPaint);
+        canvas.drawPath(mPath, mPaint);
+    }
+
+    @Override
+    public boolean onTouchEvent(MotionEvent event) {
```

Comment:
WARNING: [ClickableViewAccessibility] Custom view `com/fstyle/androidtrainning/widget/customView/TouchEventView` overrides `onTouchEvent` but not `performClick`
 If a `View` that overrides `onTouchEvent` or uses an `OnTouchListener` does not also implement `performClick` and call it when clicks are detected, the `View` may not handle accessibility actions properly. Logic handling the click actions should ideally be placed in `View#performClick` as some accessibility services invoke `performClick` when a click action should occur.